ecommerce_integrations/shopify/order.py

```diff
@@ -157,7 +157,7 @@ def get_order_taxes(shopify_order, setting):
 			{
 				"charge_type": _("On Net Total"),
 				"account_head": get_tax_account_head(tax),
-				"description": f"{tax.get('title')} - {tax.get('rate') * 100.0}%",
+				"description": f"{get_tax_account_description(tax,tax.get('title'))} - {tax.get('rate') * 100.0}%",
```

Comment:
Instead of passing the fallback text, the function can return `None` instead (if not found).

E.g. `get_tax_account_description(tax) or tax.get('title')`

Reply:
Agree... changed